source/Modules/Devon4Net.Infrastructure.AWS.CDK/Resources/Management/SecurityGroups/SecurityGroupsHandlerManager.cs

```diff
@@ -16,7 +16,12 @@ public ISecurityGroup AddSecurityGroup(string identification, string securityGro
 
         public ISecurityGroup LocateSecurityGroupById(string identification, string securityGroupId)
         {
-            return HandlerResources.AwsSecurityGroupHandler.Locate(identification, securityGroupId);
+            return HandlerResources.AwsSecurityGroupHandler.LocateById(identification, securityGroupId);
+        }
+
+        public ISecurityGroup LocateSecurityGroupByName(string identification, string securityGroupName, IVpc securityGroupVpc)
```

Comment:
Please rename the parameter "identification" to securityId as it is defined in the interface and the it is the name of the parameter on the LocateByName method